src/components/Abstractor/Abstractor.tsx

```diff
@@ -30,16 +41,30 @@ const Abstractor = ({ patientData, library, valueSetMap, questionnaire }: Props)
       console.log(lform);
       window.LForms.Util.addFormToPage(lform, 'formContainer');
 
-      // Get Questionnaire Response
-      const qr = window.LForms.Util.getFormFHIRData('QuestionnaireResponse', 'R4', lform);
-      console.log(qr);
-      //window.LForms.Util.mergeFHIRDataIntoLForms("QuestionnaireResponse", qr, lform, "R4");
     } catch (e) {
       console.error(`Error finding patient resource within bundle: ${e.message}`);
     }
   }, [patientData, library, valueSetMap, questionnaire]);
 
-  return <div id="formContainer"></div>;
+  const generateQR = () => {
+    // Generate QuestionnaireResponse
+    const qr = window.LForms.Util.getFormFHIRData('QuestionnaireResponse', 'R4');
+    console.log(qr);
+    // Signify to user that QuestionnaireResponse has been generated
+    let answerCount = 0;
+    if (qr.item) {
+      answerCount = qr.item.length;
+    }
+    setResponseGenerated(responseGenerated => ({...responseGenerated,  message: true, count: answerCount}) );
```

Comment:
not sure how this doesn't give you an error, but the setResponseGenerated function sets the state variable for responseGenerated. Meaning the argument to it should be an object that matches the `ResponseObject` interface that you defined.

```suggestion
    setResponseGenerated({ count: answerCount, isGenerated: true });
```